ssc_interface_wrapper/launch/ssc_pacmod_driver.launch

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<!--
+  Copyright (C) 2018-2019 LEIDOS.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<!--
+	This file is used to launch a CARMA compatible lidar driver for velodyne VLP-32C
+-->
+<launch>
+
+  <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" />
+  <arg name="use_kvaser" default="true" />
+  <arg name="kvaser_hardware_id" default="10376" />
+  <arg name="kvaser_circuit_id" default="0" />
+
+  <!-- Launch Wrapper -->
+  <include file="$(find ssc_interface_wrapper)/launch/ssc_interface_wrapper.launch"/>
+
+  <!-- Launch Wrapped Nodes -->
+  <!-- Drive By Wire -->
+  <include file="$(find pacmod3)/launch/pacmod3.launch" ns="pacmod">
+    <arg name="pacmod_vehicle_type" value="LEXUS_RX_450H" />
+    <arg name="use_kvaser" value="true" />
+    <arg name="kvaser_hardware_id" value="10376" />
+    <arg name="kvaser_circuit_id" value="0" />
+  </include>
+
+  <!-- SSC -->
+  <include file="$(find ssc_pm_lexus)/launch/speed_steering_control.launch">
+    <arg name="namespace" value="ssc" />
+  </include>
```

Comment:
I think we are missing the SSC interface node from Autoware. SSC Interface Wrapper node talks to SSC interface directly.